### PR TITLE
Update the flag names and default value of the dbname flag

### PR DESCRIPTION
**Pull request by blagojts**
The flag names are changed to signify that the connector is no longer 
compatible with Postgres, just TimescaleDB.

Also the deployment in the helm-chart is simplified to use
the TS_PROM environment variables to set the connection
flags.

*Warning* Upon merging, the helm chart will require a new docker build to be uploaded to the docker hub because it assumes the new flag names

### CODE REVIEW

File: pkg/pgclient/client.go [hunk 1]
```diff
@@ -30,17 +30,17 @@ type Config struct {
...
 // ParseFlags parses the configuration flags specific to PostgreSQL and TimescaleDB
 func ParseFlags(cfg *Config) *Config {
-	flag.StringVar(&cfg.host, "pg-host", "localhost", "The PostgreSQL host")
-	flag.IntVar(&cfg.port, "pg-port", 5432, "The PostgreSQL port")
-	flag.StringVar(&cfg.user, "pg-user", "postgres", "The PostgreSQL user")
-	flag.StringVar(&cfg.password, "pg-password", "", "The PostgreSQL password")
-	flag.StringVar(&cfg.database, "pg-database", "postgres", "The PostgreSQL database")
-	flag.StringVar(&cfg.sslMode, "pg-ssl-mode", "disable", "The PostgreSQL connection ssl mode")
-	flag.IntVar(&cfg.dbConnectRetries, "pg-db-connect-retries", 0, "How many times to retry connecting to the database")
+	flag.StringVar(&cfg.host, "db-host", "localhost", "The TimescaleDB host")
```
Comment (blagojts) [inline pkg/pgclient/client.go:33]: i'm not sure about this change, I wanted to signify that the connector is no longer compatible with PostgreSQL, but it is still a Postgres cluster with users, databases etc created in Postgres

File: helm-chart/templates/deployment-connector.yaml [hunk 2]
```diff
@@ -29,25 +29,21 @@ spec:
             - containerPort: 9201
               name: connector-port
           env:
-            - name: PGPORT
+            - name: TS_PROM_DB_PORT
```
Comment (cevian) [inline helm-chart/templates/deployment-connector.yaml:32]: how are these env variables read by the connector if they aren't in the args?

File: pkg/pgclient/client.go (hunk 1 above)
Comment (cevian) [reply pkg/pgclient/client.go:33]: I like this change

File: helm-chart/templates/deployment-connector.yaml (hunk 2 above)
Comment (blagojts) [reply helm-chart/templates/deployment-connector.yaml:32]: we use a package that is set up to search ENV for vars starting with TS_PROM

File: helm-chart/templates/deployment-connector.yaml [hunk 3]
```diff
@@ -29,25 +29,21 @@ spec:
...
-              -pg-ssl-mode=require,
-              -pg-host=$(PGHOST),
-              -pg-user=$(PGUSER),
-              -pg-password=$(PGPASSWORD),
-              -pg-port=$(PGPORT)
-            ]
+            - name: TS_PROM_DB_HOST
+              value: {{ tpl .Values.connection.host.nameTemplate . }}
+            - name: TS_PROM_DB_NAME
+              value: {{ .Values.connection.dbName }}
```
Comment (cevian) [inline helm-chart/templates/deployment-connector.yaml:44]: can we add the dbname to the cronjob too?

**Pull request merged**